src/blog/blogLikes.entity.ts

```diff
@@ -0,0 +1,21 @@
+import { Column, Entity, ManyToOne, PrimaryGeneratedColumn } from "typeorm";
```

Comment:
I think you don't need to create separate entities for likes and dislikes. dislike works only when user already liked post. It is used to return his/her like